### PR TITLE
Decreasing the size of our privacy warning to match designs

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Our first tailwind fix! Worked on the first try :)

### CODE REVIEW

**Comment by jonbell-lot23** [comment]
Previous design:
image

New design:
image

File: src/ui/components/shared/SharingModal/SharingModal.tsx [hunk 1]
```diff
@@ -60,13 +60,14 @@ function SharingModal({ recordingId }: PropsFromRedux) {
...
-              <strong>Important note:</strong> Replay records everything that happens in the
-              browser, including passwords you’ve typed and everything visible on the screen.{" "}
+            <div className="privacy-warning text-sm">
+              <strong>Note:</strong> Replay records everything that happens in the browser,
+              including passwords you’ve typed and everything visible on the screen.{" "}
               <a
                 target="_blank"
                 rel="noreferrer"
                 href="https://www.notion.so/replayio/Security-2af70ebdfb1c47e5b9246f25ca377ef2"
+                class="underline"
```
Comment (jasonLaster) [inline src/ui/components/shared/SharingModal/SharingModal.tsx:70]: typescript failed because `class` should be `className`

**Review by jasonLaster — APPROVED** [review]
one fix, otherwise good

File: src/ui/components/shared/SharingModal/SharingModal.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/shared/SharingModal/SharingModal.tsx:70]: Ohhhh rightttttt

**Pull request merged**